examples/musig.c

```diff
@@ -201,7 +233,7 @@ int sign(const secp256k1_context* ctx, struct signer_secrets *signer_secrets, st
         return 1;
     }
     printf("ok\n");
-    printf("Verifying signature.....");
+    printf("Verifying signature and revealed adaptor.....");
```

Comment:
I don't think we're verifying the adaptor here?

Reply:
errr extracting revealed secret, I guess is the phrase?

Reply:
I'm only saying that the printf not match the code below, which still only verifies a sig.

Reply:
oh, I added this in the wrong spot